src/Resources/config/commands.yml

```diff
@@ -1,32 +1,26 @@
 services:
+    _defaults:
+        autoconfigure: true
```

Comment:
Yes, I just approved the PR. This comment was related to my initial thought that it should be for everything.